downstream/modules/devtools/con-devtools-roles-collection-prerequisites.adoc

```diff
@@ -3,6 +3,9 @@
 [id="devtools-roles-collection-prerequisites_{context}"]
 = Prerequisites
 
+[role="_abstract"]
+You must install and configure the essential software and accounts listed in this section before you proceed.
```

Comment:
Active voice rewrite suggestion: 

"The following essential software and accounts are required to create collections: 

- {VSCode} and the Ansible extension
- Microsoft Dev Containers extension in {VSCode}
- {ToolsName}
- A containerization platform, for example Podman, Podman Desktop, Docker, or Docker Desktop
- A Red Hat account and log in access to the Red Hat container registry at registry.redhat.io. For information about logging in to registry.redhat.io, see [Authenticating with the Red Hat container registry](https://github.com/ansible/aap-docs/blob/AAP-56025-Devtools-CQA/downstream/modules/devtools/%7BURLDevelopAutomationContent%7D/installing-devtools#devtools-setup-registry-redhat-io_installing-devtools)."